src/rpc/reflect/mod.rs

```diff
@@ -103,37 +86,40 @@ pub enum Permission {
     Read,
 }
 
-/// Lotus groups methods into API versions.
-///
-/// These are significant because they are expressed in the URL path against which
-/// RPC calls are made, e.g `rpc/v0` or `rpc/v1`.
+/// Which paths should this method be exposed on?
 ///
 /// This information is important when using [`crate::rpc::client`].
-#[derive(
-    Debug,
-    Clone,
-    Copy,
-    Hash,
-    Eq,
-    PartialEq,
-    Ord,
-    PartialOrd,
-    Serialize,
-    Deserialize,
-    strum::EnumIter,
-)]
-pub enum ApiVersion {
+#[derive(Debug, Clone, Copy, Hash, Eq, PartialEq, Ord, PartialOrd)]
+pub enum ApiPaths {
+    /// Only expose this method on `/rpc/v0`
     V0,
+    /// Only expose this method on `/rpc/v1`
     V1,
+    /// Expose this method on both `/rpc/v0` and `/rpc/v1`
+    #[allow(dead_code)]
+    Both,
```

Comment:
`Both` seems less extensible when there's a future V2, how about using bit flags?

Reply:
I agree, but I would prefer to defer that to when the time comes - bitflags are slightly harder to iterate and we don't need that for now